### PR TITLE
catfish: init at 1.4.1

### PR DESCRIPTION
**Pull request by romildo**
###### Things done:
- [x] Tested using sandboxing (`nix-build --option build-use-chroot true` or [nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/search/catfish/default.nix [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{ stdenv, fetchurl, file, which, intltool, findutils, xdg_utils, pycairo,
+  gnome3, pythonPackages, wrapGAppsHook }:
+
+pythonPackages.buildPythonApplication rec {
+  majorver = "1.4";
+  minorver = "1";
+  version = "${majorver}.${minorver}";
+  name = "catfish-${version}";
+  namePrefix = "";
```
Comment (joachifm) [inline pkgs/applications/search/catfish/default.nix]: This is redundant when using `buildPythonApplication`

Comment (romildo) [inline pkgs/applications/search/catfish/default.nix]: Removed the `namePrefix = "";` line.

**Comment by joachifm** [comment]
Seems like there are missing build inputs, resulting in import errors during the test phase:
```
/tmp/nix-build-catfish-1.4.1.drv-0/catfish-1.4.1/catfish_lib/helpers.py:23: PyGIWarning: Gtk was imported without specifying a version first. Use gi.require_version('Gtk', '3.0') before import to ensure that the right version gets loaded.
  from gi.repository import Gtk, GObject
ERROR: Python module zeitgeist.client not found
ERROR: Python module zeitgeist.datamodel not found
ERROR: Python module zeitgeist not found
```

Also, there are some errors related to SQLAlchemy; no idea if that's related to this change or not.

**Comment by romildo** [comment]
Zeitgeist is an optional run time dependency for `catfish` and I have opted for omitting it. It seems that the `zeitgeist` package on the repository needs some refactoring to be detected by `catfish`, as the error persists after including `zeitgeist` in `propagatedBuildInputs`.
From the [Zeitgeist article on ArchWiki](https://wiki.archlinux.org/index.php/Zeitgeist):
> Zeitgeist is a service which logs user activities and events, anywhere from files opened to websites visited or conversations held via email and more. It makes this information readily available for other applications to use in the form of timelines and statistics.
`catfish` can be used to browse or search for recent activity using the Zeitgeist engine.

**Comment by joachifm** [comment]
Alright

**Pull request merged**